examples/tags-in-searchbox/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@algolia/autocomplete-example-tags-in-searchbox",
+  "description": "Autocomplete example with Tags in the searchbox",
+  "version": "1.2.2",
+  "private": true,
+  "license": "MIT",
+  "scripts": {
+    "build": "parcel build index.html",
+    "start": "parcel index.html"
+  },
+  "dependencies": {
+    "@algolia/autocomplete-js": "1.2.2",
+    "@algolia/autocomplete-plugin-algolia-insights": "1.2.2",
+    "@algolia/autocomplete-plugin-tags": "1.2.2",
+    "@algolia/autocomplete-theme-classic": "1.2.2",
```

Comment:
We'll need to update these deps before merging the base branch.

Reply:
Shouldn't Ship.js handle this for us?

Reply:
See https://github.com/algolia/autocomplete/pull/645#discussion_r706090313.

Reply:
Addressed in 77f0e83